### PR TITLE
Add missing include directories for ETW and HTTP client for CMake

### PR DESCRIPTION
**Pull request by ThomsonTan**
Add 2 missing include directories and also add the explicit file extension name (.cc) for `add_library()` source file.

### CODE REVIEW

File: exporters/etw/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-include_directories(include)
+include_directories(include
```
Comment (maxgolov) [inline exporters/etw/CMakeLists.txt:1]: I was thinking we'd better add this at top level since there is another module (zpages) that needs this.

Comment (maxgolov) [reply exporters/etw/CMakeLists.txt:1]: Maybe
`
# Add nlohmann/json submodule to include directories
include_directories(third_party/nlohmann-json/single_include)
`
straight in top-dir `CMakeLists.txt`. That way it'd be available to both:
- ETW exporter
- `ext/include/opentelemetry/ext/zpages/tracez_http_server.h`

I'm doing it in my PR https://github.com/open-telemetry/opentelemetry-cpp/pull/519

Comment (lalitb) [reply exporters/etw/CMakeLists.txt:1]: Agreed, we also need this for zipkin exporter.

**Review by maxgolov — APPROVED** [review]
Approved with suggestions. Use your best judgement if you add it at project level or pull it up to the top.

File: exporters/etw/CMakeLists.txt (hunk 1 above)
Comment (ThomsonTan) [reply exporters/etw/CMakeLists.txt:1]: I am slightly prefer keep it here. Moving it to top level will cause the include directory to be searched for all compilation. But it also looks good to me if some future PR moves this to the top level CMakeLists.txt.

**Pull request merged**